server/playground/index.js

```diff
@@ -1,33 +1,16 @@
-const buildSchema = require("@aerogear/data-sync-gql-core");
 const { makeExecutableSchema } = require("graphql-tools");
 const { ApolloServer } = require("apollo-server-express");
-const { log } = require("../logger");
-const models = require("../models");
-
-async function connectDataSources(dataSources) {
-    log.info("Connecting data sources");
-    Object.keys(dataSources).forEach(async key => {
-        const dataSource = dataSources[key];
-        try {
-            await dataSource.connect();
-        } catch (error) {
-            log.error(`Error while connecting datasource with key ${key}`);
-            log.error(error);
-            throw (error);
-        }
-    });
-}
-
+const { Core } = require("@aerogear/data-sync-gql-core");
+const config = require("../config").postgresConfig;
 
 async function ApplyExecutableMiddleware(app) {
-    // read default schema
-    // Attach middleware to app
-    const { schema, dataSources } = await buildSchema(models, null, null, makeExecutableSchema);
+    const core = new Core(config, makeExecutableSchema);
+    const { schema, dataSources } = await core.buildSchema("default");
     const apolloServer = new ApolloServer({
         schema
     });
     apolloServer.applyMiddleware({ app, path: "/graphqldata" });
-    await connectDataSources(dataSources);
+    await core.connectActiveDataSources(dataSources);
```

Comment:
No strong opinion here - more asking question: Do you think we should wait for all datasources to connect before we will enable middleware?